projects/@shared/components/properties-item-new.vue

```diff
@@ -95,6 +95,11 @@
       </div>
     </dd>
 
+    <dd v-else-if="valueExceedsLimit" class="properties-item__value">
+      <span>{{ splitValue.start }}</span>
+      <span style="text-decoration: line-through; opacity: .5;">{{ splitValue.end }}</span>
```

Comment:
Dit is niet dynamisch, naar mijn mening moet het gewoon een `class` zijn.

Reply:
Changed.